Include/RmlUi/Core/Lua/LuaType.inl

```diff
@@ -176,7 +176,8 @@ int LuaType<T>::gc_T(lua_State* L)
     if(lua_istable(L,-1) ) //[-1 = 2], if it is a table
     {
         char name[max_pointer_string_size];
-        tostring(name, max_pointer_string_size, obj);
+        void* ptrHold = lua_touserdata(L,1);
```

Comment:
Should we validate *ptrHold == obj?

Reply:
Well that's what `check` does - get ptrHold and dereference it.